### DISCUSSION TITLE
Cannot delete repo

### DISCUSSION THREAD

**Discussion by yihao5un** [community/community#136613] [Repositories] [answered]
### Select Topic Area

Question

### Body

I want to delete the repo from the left repo list
![image](https://github.com/user-attachments/assets/8bc9431b-9222-4013-8de5-8cfb86279592)
but when i cannot find and delete it in my repos
![image](https://github.com/user-attachments/assets/4d2c6abe-91c5-45b3-ab22-b9194da7f133)

**Comment by shubhamdevgupta** [comment]
Basically this is not your repo, this is forked repo and the owner can only delete this repo

**Reply by yihao5un** [reply]
Thanks, but I don't want this forked repo show in the left list, what should I do?

**Comment by MiguelCardoso19** [comment]
Since its a forked repo you need to delete the fork not the repo.

**Reply by yihao5un** [reply]
Thanks, but I don't want this forked repo show in the left list, what should I do?

**Comment by amirallami-code** [comment]
hello, the repository you showed is not your own and therefore you can not delete it completely, you can delete the fork you made, for this action you can go to the forked repository settings and go to the very bottom and press the "delete repository" button, if you do not want to delete the forked repository there is no way it will be deleted from the left list, even if you change the visibility it will be shown there too I think

**Comment by yihao5un** [comment]
Hi, i do want to delete this forked repo, but i can not find the settings button when i click the repo
![image](https://github.com/user-attachments/assets/d5885ad6-4ae4-482b-9e3b-323485d60583)
![image](https://github.com/user-attachments/assets/3a3da93d-d378-49e8-8ea5-f33dc7154caf)

**Reply by amirallami-code** [reply]
i think got it, this is not forked its just the main repository i think its there because you have impression with that repo

**Reply by yihao5un** [reply]
Yes, actually i issued that repo, then i found the repo always show in the left repo page... i really do not how to delete it ..

**Reply by amirallami-code** [reply]
In that scenario I think you will be not able to delete that from the left list, you can just make more repositories and make more impressions with other repos so that it comes downer and downer in the list and gets finally deleted

**Reply by yihao5un** [reply]
Thanks, I will try it!

**Reply by amirallami-code** [reply, marked as answer]
If you could accept my answer, that would be much appreciated. Thank you.

**Comment by shubhamdevgupta** [comment]
Hey @yihao5un  if you want to deleted forked repo you just need to goo there 
![C9F03D00-A3BE-45E4-84FB-ABBFEB0D1972](https://github.com/user-attachments/assets/49043fb8-5de6-4e2c-8b38-392e32de6a07)
![6843A5A2-71EB-4DA6-B959-7239FD0B9831](https://github.com/user-attachments/assets/ca770326-d0b3-4cd4-a277-f5e5ae945c27)

and then scrool down click on delete  repository and enter your github password this fork repository will gone delete

**Reply by shubhamdevgupta** [reply]
If you could accept my answer, that would be much appreciated. Thank you.

**Comment by jrucker-amd** [comment]
I created a repo, and it shows "This is an empty repository". However, there is no option to upload files, I don't have permission to push (even with push -f), and there isn't even the usual "Settings" button in order to delete the repo.
